popx/migrator_test.go

```diff
@@ -58,6 +58,8 @@ func TestMigratorUpgrading(t *testing.T) {
 
 	for name, c := range connections {
 		t.Run(fmt.Sprintf("database=%s", name), func(t *testing.T) {
+			t.SkipNow()
```

Comment:
This test does not actually test whether the schemas are the same. Dumping of schemas fails because of various reasons (e.g. the statement changed in CRDB, sqlite tries to use the `sqlite` binary). It probably passed until now because the dump function just returned `nil` right away.
In the very end of this test, where we have `assert.EqualValues(t, expected, actual)`, we don't compare the schema dumps, but the errors returned by dump. Therefore I chose to skip the test and not fix it, as it is fundamentally flawed and did not work in the first place.